akkaConstruct/src/test/scala/akka/contrib/construct/ActorOfSpec.scala

```diff
@@ -134,6 +149,17 @@ object ActorOfSpec {
   }
 
   object Child {
-    val constructor: Constructor[Seq[String], Child] = Constructor(new Child(_))
+    class Construct extends P1ConstructActor with DefaultConstructArgs {
+      override type ActorClass = Child
+      override type P1 = Seq[String]
+      override def actorOf(path: Seq[String])(implicit tag: ClassTag[Child], params: ConstructActorParams): ActorOf[Child] =
+        ActorOf(construct(path))
+      override protected def construct(path: Seq[String])(implicit params: ConstructActorParams): Child = new Child(path)
+      override protected val defaultArgs: Seq[String] = Seq()
```

Comment:
@jonhlee this is the way to do CustomActorOf for future reference. You create a class like this and reference this class in your constructor in order to create custom of ActorOf instances.